service/src/test/resources/ehr/mapper/non_consultation_bundles/allergy-intolerance.xml

```diff
@@ -0,0 +1,41 @@
+<ehrComposition classCode="COMPOSITION" moodCode="EVN">
```

Comment:
Is this ehrComposition being generated from the AllergyIntolerance itself ? If so it's incorrect - AllergyIntolerance should only generate the CompoundStatement structure with nested ObservationStatement.

Reply:
it is, this will need to be fixed in another story.